Misc/NEWS.d/next/Tests/2020-06-23-12-02-45.bpo-41085.JZKsyz.rst

```diff
@@ -0,0 +1 @@
+Fix integer overflow in the array.array.index() method on 64-bit Windows for index larger than 2**31.
```

Comment:
I suggest the following change to enhance the format (add a link on the method):

```suggestion
Fix integer overflow in the :meth:`array.array.index` method on 64-bit Windows
for index larger than ``2**31``.
```